tests/phpunit/test-wc-stripe-webhook-state.php

```diff
@@ -72,6 +72,15 @@ public function tearDown() {
 		delete_option( WC_Stripe_Webhook_State::OPTION_TEST_LAST_ERROR );
 	}
 
+	private function cleanup_webhook_secret () {
+		$stripe_settings = get_option( 'woocommerce_stripe_settings', array() );
+		unset( $stripe_settings['webhook_secret'] );
+		unset( $stripe_settings['test_webhook_secret'] );
+		unset( $stripe_settings['testmode'] );
```

Comment:
I'd remove this `unset( $stripe_settings['testmode'] );` because this is already being done in `tearDown`.

And just a small tweak in `cleanup_webhook_secret ()` with the extra space.

(We are almost done with a PHPCS PR, so we don't need to worry about coding standards 🙌)

Reply:
Thanks for spotting this! Indeed, it's hard to notice without linter in place. Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1493/commits/0c6bb63824fa7c31981363564551ee5e6e5642fb

Reply:
btw, I've noticed linter in GitHub actions checks but it's not available locally. Is it the case? If it's available would be nice to have an npm command and pre commit hook to lint the changes.

Reply:
@vbelolapotkov we had the [PHP CodeSniffer PR](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1445) merged yesterday. `npm lint:php` and `npm lint:php-fix` were added. You probably need `composer install` and `npm run up` again next time.

Reply:
Fantastic, thank you! I haven't made a rebase and so haven't seen them :)